dashboard/app/controllers/pd/workshop_dashboard_controller.rb

```diff
@@ -3,11 +3,18 @@ class WorkshopDashboardController < ApplicationController
     before_action :authenticate_user!
 
     def index
-      unless current_user.admin? ||
-        current_user.district_contact? ||
-        current_user.workshop_organizer? ||
-        current_user.facilitator?
+      @permission =
+        if current_user.admin?
+          :admin
+        elsif current_user.workshop_organizer?
+          :workshop_organizer
+        elsif current_user.facilitator?
+          :facilitator
+        else
+          nil
+        end
 
+      unless @permission
         render_404
         return
```

Comment:
Tabbing here seems weird, not sure if its the way this is rendering in GitHub though